src/Core/src/Platform/iOS/IIosApplicationDelegateHandler.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Foundation;
+using UIKit;
+
+namespace Microsoft.Maui
+{
+	/// <summary>
+	/// Allow to get iOS UIApplication lifecycle callbacks.
+	/// </summary>
+	public interface IIosApplicationDelegateHandler : IPlatformLifecycleHandler
```

Comment:
For the case on iOS where there are registrations that may cause App Store warnings or rejections, we will have to come up with some way to actually remove overrides (hard) or override the "responds to selector" method and use reflection to detect things and say no.

Reply:
And, we might need to add those methods here for the previews and fix them up later.